packages/core/src/browser/frontend-application.ts

```diff
@@ -255,6 +267,8 @@ export class FrontendApplication {
                     }, transitionDuration);
                 });
             });
+            return this.stopwatch.startAsync('revealShell', 'Finished loading frontend application', doRevealShell,
+                { defaultLogLevel: LogLevel.INFO });
```

Comment:
The problem here is that we now log the time it takes to reveal the shell which is a predetermined animation. So the message no longer fits as it's "Finished loading + animation time".
```suggestion
            return this.stopwatch.startAsync('revealShell', 'Replace loading indicator with actual UI (animation)', doRevealShell,
                { defaultLogLevel: LogLevel.INFO });
```

Reply:
Maybe we should add more log messages to the `start` method?